core/kazoo_csv/download_csvs.py

```diff
@@ -1,19 +0,0 @@
-#!/usr/bin/env python2
-
-import requests, sys, json
-
-print("version: ", requests.__version__)
-csvs_url = 'https://api.github.com/repos/maxogden/csv-spectrum/contents/csvs'
```

Comment:
I disagree with removing this script/step from testing. Just dumping the CSVs into the repo removes attribution from where they came from.